src/Traits/SoftekPropertySetterTrait.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace DexproSolutionsGmbh\SoftekBarcodeWrapper\Traits;
+
+/**
+ * This trait should include all necessary set functions within the softek barcode toolkit
+ * you can include this class in your API but be sure that the class has a valid pointer to an barcode instance and the
+ * loaded ffi library.
+ */
+trait SoftekPropertySetterTrait
```

Comment:
As discussed, this trait is an abstraction that seems a bit early. Let's revert this and inline `$this->ffi->mtSetMultipleRead($this->instance, $bool);` into the `SoftekBarcodeAPI`.